artifacts/deploy/bootstrap-token-configuration.yaml

```
@@ -59,296 +59,90 @@ subjects:
 
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+  name: system:certificates.k8s.io:certificatesigningrequests:agent
```

```suggestion
metadata:
  labels:
    karmada.io/bootstrapping: rbac-defaults
  name: system:karmada.io:certificatesigningrequester
```
1. Remove the annotation `rbac.authorization.kubernetes.io/autoupdate: "true"` because it is used to enable `Kubernetes default role` reconcile, not for the Karmada default role.
2. Rename labels with karmada.io prefix
3. Rename the default role name 

What do you think?